Stripe/STPPaymentContext.m

```diff
@@ -292,9 +293,11 @@ - (void)presentPaymentMethodsViewControllerWithNewState:(STPPaymentContextState)
             paymentMethodsViewController.prefilledInformation = self.prefilledInformation;
             paymentMethodsViewController.paymentMethodsViewControllerFooterView = self.paymentMethodsViewControllerFooterView;
             paymentMethodsViewController.addCardViewControllerFooterView = self.addCardViewControllerFooterView;
+            paymentMethodsViewController.navigationItem.largeTitleDisplayMode = self.largeTitleDisplayMode;
 
             UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:paymentMethodsViewController];
             navigationController.navigationBar.stp_theme = self.theme;
+            navigationController.navigationBar.prefersLargeTitles = YES;
```

Comment:
Based on the docs, it looks like it isn't necessary to expose this property on `PaymentContext` too – just setting it to always `YES` always seems fine.

> When this property is set to true, the navigation bar allows the title to be displayed out-of-line and using a larger font. The navigation item used to build the bar must specify whether it wants its title displayed in the large or small format. Use the largeTitleDisplayMode property to configure the title's appearance.

> When the property is set to false, the navigation bar displays the title inline with the other bar button items.